README.md

```diff
@@ -50,10 +50,6 @@ The project is split up in a number of subprojects:
 Additionally, 'plugin-tester-java' and 'plugin-tester-scala' contain an example
 project in Java and Scala respectively, with both sbt and Gradle configurations.
 
-## Compatibility & support
-
-If used with JDK 8 prior to version 1.8.0_251 you must add an [ALPN agent](https://doc.akka.io/docs/akka-http/10.1/server-side/http2.html#application-layer-protocol-negotiation-alpn-).
```

Comment:
the link does not work with Pekko docs because that subsection was removed from Akka docs before we forked - seems tidiest just to remove this

Reply:
Rather than removing it can we instead just comment that you need to use JDK 8 1.8.0_251 or later?

Reply:
I've added this back but highlight the link is an Akka link.

Reply:
I have a preference for JDK 8 1.8.0_251 or later because I don't want to encourage people using older JDK's and as a bonus we don't need to link to any Akka docs.

Reply:
this same stuff is basically in the paradox docs - I'd prefer to keep them consistent

Reply:
Okay fine, I won't die on this hill. Will approve.